sig-scalability/throughput_burts_slo.md

```diff
@@ -0,0 +1,23 @@
+# SLO: Kubernetes cluster of size at least X is able to start Y Pods in Z minutes
+
+## Burst Pod Startup Throughput SLO
+### Summary
+Kubernetes cluster of size at least X is able to start Y Pods in Z minutes.
+### User Stories
+- User is running a workload of X total pods and wants to ensure that it can be started in Y time. 
+- User wants to understand how quickly they can react to a dramatic change in workload profile - e.g. they need to begin scheduling new pods when QPS meets Z% of maximum throughput of users system - to accommodate a spike due to a Black Friday Sale.
+- User is running a huge serving app on a huge cluster. He wants to know how quickly he can recreate his whole setup in case of a serious disaster which will bring the whole cluster down.
+
+Current steady state SLOs are do not provide enough data to make these assessments about burst behavior. 
+## SLO definition (full)
+### Test setup
+Standard performance test kubernetes setup.
```

Comment:
We should have a link to the doc containing this here (and we already have this doc - this is your previous one, even though there are multiple TODOs there).